src/components/ChatItem/index.ts

```diff
@@ -0,0 +1,68 @@
+import Block from 'core/block/Block';
+import './chatItem.css';
+
+export interface ChatItemProps {
+  id: number;
+  title: string;
+  last_message: {
+    user: {
+      first_name: string;
+      second_name: string;
+      avatar: string;
+      email: string;
+      login: string;
+      phone: string;
+    },
+    time: string;
+    content: string;
+  }
+  unread_count: string;
+  avatar: string | null;
+  activeClassName: string;
+}
+
+// TODO active class name for chat
+class ChatItem extends Block<ChatItemProps> {
+  static componentName = 'ChatItem';
+
+  constructor({
+    id,
+    title,
+    avatar,
+    last_message,
+    unread_count,
+    activeClassName,
+  }: ChatItemProps) {
+    super({
+      id,
+      title,
+      last_message,
+      unread_count,
+      avatar,
+      activeClassName,
+    });
+  }
```

Comment:
Если никаких преобразований пропсов нет, то можно короче
```suggestion
  constructor(props: ChatItemProps) {
    super(props);
  }
```
Вообще можно не определять конструктор, если он содержит только вызов родительского

Reply:
ок